packages/ez-react/src/docs/4_2_responsiveCharts.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Meta } from '@storybook/addon-docs';
+import Tabs from './Tabs';
+
+<Meta
+  title="Get Started/Essentials/Responsive Charts"
+  parameters={{ options: { isToolshown: false }, id: 5 }}
+/>
+
+# Responsive Charts
+
+All chart recipes in **EazyChart** comes with fixed width,
```

Comment:
All chart recipes in **EazyChart** come with fixed width,

Reply:
Thanks! done!